internal/types/action.go

```diff
@@ -87,7 +88,9 @@ func (a *Action) WriteDocumentation(inputTable, outputTable *strings.Builder) er
 		output = utils.ReplaceBytesInBetween(input, inputStartIndex, inputEndIndex, []byte(inputsStr))
 	} else {
 		inputsStr := fmt.Sprintf("%s\n\n%v", internal.InputsHeader, inputTable.String())
-		output = bytes.Replace(input, []byte(internal.InputsHeader), []byte(inputsStr), -1)
+		if inputTable.String() != "" {
```

Comment:
An alternative here would be to remove the redundant initialization of `inputsStr` and move it above and call `strings.TrimSpace`  passing the value  of `inputsStr`